### DISCUSSION TITLE
82

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#95] [General]
# 潮流周刊第82期 - 湖州

照片拍于端午去湖州玩了 3 天，休闲休闲，爬了一个很久的山看了看全貌，整体而言浙江很多小城市的布局都很整齐的样子。

https://weekly.tw93.fun/posts/82-%E6%B9%96%E5%B7%9E

**Comment by eshitaislamjula** [comment]
偶然发现这个周刊，翻看了好多期哈哈，内容很棒，字体也好看！
